pkg/aws/helpers.go

```diff
@@ -819,3 +819,16 @@ func GetManagedPolicyARN(policies map[string]*cmv1.AWSSTSPolicy, key string) (st
 
 	return policy.ARN(), nil
 }
+
+// GetAccountRolePolicyKeys returns the policy key for fetching the managed policy ARN
+func GetAccountRolePolicyKeys(roleType string) []string {
+	if roleType == InstallerAccountRole {
```

Comment:
Will this be deleted when we get full aws support for managed policies?

Reply:
No, the mapping between the keys and the ARNs are coded in the backend.
So the CLI fetches the ARNs from the backend, and the keys will be the same.